lib/housing-wanted/housing-wanted.models.js

```diff
@@ -0,0 +1,45 @@
+const Joi = require('joi');
+
+module.exports = {
+  db: (sequelize, Sequelize) => {
+    const HousingWanted = sequelize.define('HousingWanted', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      beds: Sequelize.INTEGER,
+      baths: Sequelize.DECIMAL,
+      contact: Sequelize.STRING,
+      body: Sequelize.TEXT,
+      status: Sequelize.STRING,
+      details: {
+        type: Sequelize.JSONB,
+        defaultValue: {},
+        "cost": {
+          value: Sequelize.DECIMAL,
+        },
+        "security deposit": {
+          value: Sequelize.INTEGER,
+        },
+        "application fee": {
+          value: Sequelize.INTEGER,
+        },
+        "furnished": {
+          value: Sequelize.BOOLEAN,
+        },
+
+      },
+    });
+
+    HousingWanted.associate = function (models) {
+      models.HousingWanted.hasMany(models.PostalAddress);
```

Comment:
please remove the association to PostalAddress. The user doesn't specify addresses for the housing they want. Otherwise they could just reach out to those properties :P